Trumpf.Coparoo.Web/Controls/Span.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Trumpf.Coparoo.Web.Controls
+{
+    using OpenQA.Selenium;
+
+    /// <summary>
+    /// Span control object.
+    /// </summary>
+    public class Span : ControlObject
+    {
+        /// <summary>
+        /// Gets the search pattern.
+        /// </summary>
+        protected override By SearchPattern => By.TagName("span");
+
+        /// <summary>
+        /// Gets the text.
+        /// </summary>
+        public string Text => Node.Text;
```

Comment:
For a general `span`-element I wouldn't expect a `Text` as it can contain essetially anything.